contracts/cosmwasm-vm/cw-xcall-ibc-connection/src/admin.rs

```diff
@@ -89,9 +89,9 @@ impl<'a> CwIbcConnection<'a> {
         &self,
         store: &mut dyn Storage,
         info: MessageInfo,
-        new_admin: String,
+        new_admin: Addr,
     ) -> Result<Response, ContractError> {
-        if new_admin.is_empty() {
+        if new_admin.to_string().is_empty() {
```

Comment:
i think its better to do validate_address as well while setting new value. String can be non empty but invalid here.

Reply:
This method is only used in the test, and this PR #692 suggests the removal of this function. If not removed, will update there.